scripts/globals/mobs.lua

```diff
@@ -66,7 +68,7 @@ function phOnDespawn(ph,phList,chance,cooldown)
                 DisallowRespawn(phId, true);
                 DisallowRespawn(nmId, false);
                 UpdateNMSpawnPoint(nmId);
-                nm:setRespawnTime(GetMobRespawnTime(phId));
+                nm:setRespawnTime(immediate and 1 or GetMobRespawnTime(phId));
```

Comment:
This is a little confusing to read. I never wanna have to think about what order lua handles its and's and or's again. Pls make explicit for the next guy, thx.